src/StatusQ/Components/StatusListItem.qml

```diff
@@ -326,8 +326,10 @@ Rectangle {
                     contentWidth: row.width
                     padding: 0
                     clip: true
+                    interactive: false
```

Comment:
should this be fixed false or could be "parametrizable"?

Reply:
hey there is currently no use case where this needs to be interactive, this was forgotten to be done earlier.